ci/update-requirements.py

```diff
@@ -20,7 +20,7 @@
 # Package dependencies can vary depending on the Python version.
 # We thus have to run pip-compile with the lowest Python version that
 # the project supports.
-EXPECTED_PYTHON_VERSION = (3, 6)
+EXPECTED_PYTHON_VERSION = (3, 8)
```

Comment:
It should be python3.7

Reply:
I think it will be easier to use 3.8 as it is default for ubuntu 20, otherwise for setup ci environment we need manually install python3.7

Reply:
ok